atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolingContainer.java

```diff
@@ -341,13 +343,22 @@ private NonEvictionLoggingEvictionPolicy(EvictionPolicy<T> delegate) {
         public boolean evict(EvictionConfig config, PooledObject<T> underTest, int idleCount) {
             boolean delegateResult = delegate.evict(config, underTest, idleCount);
             // PDS-146088: the issue manifests with failures to evict anything
-            if (!delegateResult && log.isDebugEnabled()) {
-                log.debug(
-                        "Considered an object to be evicted from the Cassandra client pool, but did not evict it",
-                        SafeArg.of("underTestState", underTest.getState()),
-                        SafeArg.of("idleState", underTest.getIdleTimeMillis()),
-                        SafeArg.of("idleCount", idleCount),
-                        SafeArg.of("evictionConfig", config));
+            if (log.isDebugEnabled()) {
+                if (delegateResult) {
+                    log.debug(
+                            "Attempting to evict an object from the Cassandra client pool",
```

Comment:
This could be happening on every run, but it fails to evict, so I thought it was worth logging (we've actually validated that the other log line is _not_ logged).

Reply:
I'm ok with stacks that are turning into a fire pit to turn into _more_ of a fire pit here anyway.